Scripts/Scenes/Lobby/SceneLobby.cs

```diff
@@ -130,18 +119,14 @@ public void AddPlayer(uint id, string name)
             player.SetId(id);
         }
 
-        public void RemovePlayer(uint id)
+        public async void RemovePlayer(uint id)
         {
             if (UIPlayers.DoesNotHave(id))
                 return;
 
             if (NetworkManager.IsHost)
             {
-                Task.Run(async () => {
-                    await WebClient.Post("servers/players/remove", new Dictionary<string, string> {
-                        { "Ip", WebClient.ExternalIp }
-                    });
-                });
+                await WebClient.RemovePlayerAsync();
```

Comment:
`await WebClient.RemovePlayerAsync();` should be moved ahead of `LobbyNotifier.Print("Game start was cancelled");`